sharkfuser/CMakeLists.txt

```diff
@@ -48,27 +48,24 @@ list(APPEND CMAKE_MODULE_PATH
 include(CTestMacros)
 include(SharkfuserUtils)
 
-# TODO: IREE's CMake Config files---`IREECompilerConfig.cmake`, and
-# `IREERuntimeConfig.cmake`---are both packaged into `lib/cmake/IREE`, but
-# CMake's `find_package` defaults to looking in `lib/cmake/IREECompiler` and
-# `lib/cmake/IREERuntime` respectively. If IREE packaged its CMake Config files
-# in the expected locations, and a user installed to `/usr/local/lib`, we
-# wouldn't need to set a CMAKE_PREFIX_PATH like we do below.
-
-# Add pre-built IREE install location to CMAKE_PREFIX_PATH
-set(IREE_CMAKE_PATH "/usr/local/lib/cmake/IREE")
-if(EXISTS ${IREE_CMAKE_PATH})
-  list(APPEND CMAKE_PREFIX_PATH ${IREE_CMAKE_PATH})
-else()
-  message(WARNING "IREE not installed at '${IREE_CMAKE_PATH}', IREE path must be "
-  "set through -DCMAKE_PREFIX_PATH or -DIREECompiler_DIR + -DIREERuntime_DIR.")
-endif()
-
 # Find IREE packages + tools
-find_package(IREECompiler REQUIRED)
+# It is up to the consumer of Fusilli to determine how to bring the IREE dependencies in,
+# but here's our recommendation:
+# - Get CLI tools (iree-compile, iree-opt) from python packages
+# - For C-API interfacing with IREERuntime, have it built from source and statically linked
+#
+# Once IREERuntime is built, just point to the build location with -DIREERuntime_DIR or
+# -DCMAKE_PREFIX_PATH, like so:
+#
+#     -DIREERuntime_DIR=</path/to/iree/build/lib/cmake/IREE>
```

Comment:
The recommended way to use the IREE runtime is via the low level C API, depending on the headers and functions that you need _in your application_: https://iree.dev/reference/bindings/c-api/#runtime-api. This external build in a Linux-only dockerfile (https://github.com/sjain-stanford/docker/blob/cc39889895a2d0e2187d3135d037e7c172ec7263/Dockerfile#L49-L68) hides relevant details from developers like compilation options used and which source files are needed.